app/api/temp_messages/messages/create.rb

```diff
@@ -0,0 +1,26 @@
+module TempMessages
+  module Messages
+    class Create < Grape::API
+      desc 'Creates a spline that can be reticulated.'
+      params do
+        requires :message, type: Hash do
+          requires :content, type: String, desc: 'Content of the message.'
+          requires :expiration_limit,
+                   type: Integer,
+                   desc: 'Amount, in terms of the expiration type, that the message will be available.'
+          requires :expiration_type,
+                   type: String,
+                   desc: 'Type of expiration.',
+                   values: %w[hour hours day days week weeks month months visit visits]
```

Comment:
instead of leaving these values hardcoded, can we use the ones defined in the enum in the model?

Reply:
Done!